src/content/en/updates/2019/06/image-support-for-async-clipboard.md

```diff
@@ -180,15 +236,54 @@ document.addEventListener('copy', async (e) => {
 });
 ```
 
+### Demo
+
+<iframe style="border: solid 1px;" width="100%" height="500"
+  src="https://jsfiddle.net/0794oysr/2/embedded/result,js,html,css"
+  allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"
+></iframe>
+
+### Security
+
+Opening up the Asynchronous Clipboard API for images comes with certain
+[risks](https://w3c.github.io/clipboard-apis/#security) that need to be carefully evaluated.
+One new challenge are so-called [image compression bombs](https://bomb.codes/bombs#images),
```

Comment:
More serious than large images are specifically-crafted malicious images that are designed to exploit known vulnerabilities in the native OS. This is why we can't just copy the image directly to the native clipboard, and why we require (in Chrome) that the image be transcoded.

Reply:
Thanks, I have added this fact almost word for word.